beacon_node/beacon_chain/tests/store_tests.rs

```diff
@@ -3337,6 +3348,9 @@ async fn deneb_prune_blobs_margin_test(margin: u64) {
         Some(deneb_fork_slot)
     );
     check_blob_existence(&harness, Slot::new(1), harness.head_slot(), true);
+    if check_data_columns {
+        check_data_column_existence(&harness, Slot::new(1), harness.head_slot(), true);
+    }
```

Comment:
Same as above, when `FORK_NAME=fulu`, `check_blob_existence` would fail already before hitting this line.